tests/test_isamples_metadata.py

```diff
@@ -67,6 +77,27 @@ def test_sesar_dicts_equal(sesar_source_path, isamples_path, timestamp):
     )
 
 
+@pytest.mark.parametrize(
+    "sesar_source_path,isamples_path,timestamp", SESAR_test_values
+)
```

Comment:
We actually didn't have any tests that took a Thing and transformed it to a SOLR document.  We do now…